src/main/java/org/zalando/nakadi/security/ClientResolver.java

```diff
@@ -15,9 +15,8 @@
 import org.zalando.nakadi.util.FeatureToggleService;
 
 import java.security.Principal;
-import java.util.Collections;
+import java.util.Map;
 import java.util.Optional;
-import java.util.Set;
 
 import static org.zalando.nakadi.config.SecuritySettings.AuthMode.OFF;
 
```

Comment:
could you please remove the injection of unused FeatureToggleService? thank you

Reply:
Just for the record, it was removed 👍